source/content/guides/decoupled-sites/02-considerations.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Pantheon Decoupled
+subtitle: Considerations
+description: What you need to know to get started with Pantheon Decoupled.
+categories: [platform]
+tags: [webops, workflow, decoupled]
+contributors: [joa-pan]
+type: guide
+layout: guide
+showtoc: true
+anchorid: considerations
+permalink: docs/guides/decoupled-sites/considerations/
+editpath: decoupled-sites/02-considerations.md
+reviewed: "2022-07-31"
+---
+
+You need to ensure you system has the correct components to use Pantheon Decoupled. 
+
+## Components for Pantheon Decoupled
+
+The following components are needed to run Pantheon Decoupled: 
+
+* NodeJS: Installing NVM using Homebrew is recommeded for Mac users.
+
+* Lando: Install the latest release of Lando. Lando ships with a recommended version of Docker Desktop if you do not already have it installed.
```

Comment:
Is this required or optional? Lando templates - are they still supported or was this just for development  @CobyPear   @backlineint